File: minMaxSum/minMax.py
#!/bin/python3

import math
import os
import random
import re
import sys

# Complete the miniMaxSum function below.
def miniMaxSum(arr):
    sumArr = sum(arr)
    arrMax = max(arr)
    arrMin = min(arr)
    print (sumArr - arrMax, sumArr - arrMin)

if __name__ == '__main__':
    arr = [5,5,5,5,5]

    miniMaxSum(arr)

File: minMaxSum/minMax.c
#include <assert.h>
#include <limits.h>
#include <math.h>
#include <stdbool.h>
#include <stddef.h>
#include <stdint.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

char* readline();
char** split_string(char*);

// Complete the miniMaxSum function below.
void miniMaxSum(int arr_count, int* arr) {
    int tot = 0;
    int max = 0;
    int min = 5;

    for (int i = 0; i < 5; ++i) {
        tot += arr[i];
        if (arr[i] > max) max = arr[i];
        if (arr[i] < min) min = arr[i];
    }
    printf("%d %d", tot-max, tot-min);
}

int main()
{

    int arr[5] = {1,2,3,4,5};

    int arr_count = 5;

    miniMaxSum(arr_count, arr);

    return 0;
}

